new_buildpack_test.go

```diff
@@ -0,0 +1,89 @@
+package pack_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/heroku/color"
+	"github.com/pelletier/go-toml"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/internal/dist"
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+func TestNewBuildpack(t *testing.T) {
+	color.Disable(true)
+	defer color.Disable(false)
+	spec.Run(t, "new_buildpack", testNewBuildpack, spec.Parallel(), spec.Report(report.Terminal{}))
+}
+
+func testNewBuildpack(t *testing.T, when spec.G, it spec.S) {
+	var (
+		subject *pack.Client
+		tmpDir  string
+	)
+
+	it.Before(func() {
+		var err error
+
+		tmpDir, err = ioutil.TempDir("", "new-buildpack-test")
+		h.AssertNil(t, err)
+
+		subject, err = pack.NewClient()
+		h.AssertNil(t, err)
+	})
+
+	when("#NewBuildpack", func() {
+		it("should create bash scripts", func() {
```

Comment:
Can you add a few other tests? Specifically, it would be great to formalize through tests what the behavior will be if you run this in a directory where some/all of the files already exist (I'd at the least expect a clear error, and definitely for it not to clobber the files)

Reply:
Addressed in c288ade